src/os/bluestore/BlueStore.cc

```diff
@@ -36,6 +36,8 @@
 #include "common/PriorityCache.h"
 #include "Allocator.h"
 #include "FreelistManager.h"
+#include "ZonedAllocator.h"
```

Comment:
please guard this with `#ifdef HAVE_LIBZBD`.

Reply:
Done.

Reply:
please note there are more places need to be guarded. i just pointed one of them.

Reply:
Ugh, right.

Reply:
@tchaikov Yay, it passed! Correct me if I'm wrong, but it looks like the issue was include and not what https://github.com/ceph/ceph/pull/40873 fixed.

@ifed01 You've reviewed this before, so I think you can just approve now.